lib/beaker/dsl/helpers/host_helpers.rb

```diff
@@ -508,6 +504,19 @@ def create_tmpdir_on(host, path_prefix = '', user=nil)
           end
         end
 
+        # 'echo' the provided value on the given host
+        # @param [Host] host The host to execute the 'echo' on
+        # @param [String] val The string to 'echo' on the host
+        # @return [String] The echo'ed value
+        def echo_on_host host, val
```

Comment:
I believe to differentiate between it and host.echo.  I still believe that this can be taken on in a separate bug as the issue at hand in this PR is not that code chunk itself (which is just moving from one file to another) but the removal of 'type'.